libafl_qemu/src/qemu/mod.rs

```diff
@@ -570,6 +572,20 @@ impl From<u8> for HookData {
 
 #[allow(clippy::unused_self)]
 impl Qemu {
+    #[allow(clippy::must_use_candidate)]
+    pub fn with_options(
```

Comment:
I _feel_ like the options could/should be a `Builder`? like Qemu::build().set_foo().build()